scripts/cat/pelts.py

```diff
@@ -270,13 +283,17 @@ def __init__(
                     break
```

Comment:
```suggestion
                    continue
```
`break` here prevented newborn code from running sometimes.